SECURITY.md

```diff
@@ -0,0 +1,60 @@
+# Security Process and Policy
+
+This document provides the details on the Veriason security policy and details the processes
+surrounding security handling including a how to guide on reporting a security vulnerability
+for anything within the Veraison organization.
+
+## Report A Vulnerability
+
+We’re extremely grateful for security researchers and users who report vulnerabilities
+to the Veraison community. All reports are thoroughly investigated by a set of Veraison maintainers.
+
+To make a report please email the private security list at ___@___ with the details.
+
+### When To Send A Report
+
+You think you have found a vulnerability in a Veraison project or a dependency of a Veraison project. This can be any of the repositories on the [Veraison GitHub organization](https://github.com/veraison).
+
+### Security Vulnerability Response
+
+Each report will be reviewed and receipt acknowledged within 3 business days. This will set off the security review process detailed below.
+
+Any vulnerability information shared with the security team stays within the Veraison project and will not be shared with others unless it is necessary to fix the issue. Information is shared only on a need to know basis.
+
+We ask that vulnerability reporter(s) act in good faith by not disclosing the issue to others. And we strive to act in good faith by acting swiftly, and by justly crediting the vulnerability reporter(s) in writing.
+
+As the security issue moves through triage, identification, and release the reporter of the security vulnerability will be notified. Additional questions about the vulnerability may also be asked of the reporter.
```

Comment:
Additional questions about the vulnerability map also be asked from the reporter